pkgs/top-level/all-packages.nix

```
@@ -13410,6 +13410,8 @@ with pkgs;
 
   vpn-slice = python3Packages.callPackage ../tools/networking/vpn-slice { };
 
+  vpuppr-bin = callPackage ../applications/graphics/vpuppr-bin { };
```

```suggestion
  vpuppr = callPackage ../applications/graphics/vpuppr { };
```
Please use [source provenance](https://github.com/NixOS/nixpkgs/blob/master/doc/stdenv/meta.chapter.md#source-provenance-sec-meta-sourceprovenance) instead.

---

I accidentally squashed the commit without thinking - you can find the changes in my forced update under line 109 of the init commit.